### PR TITLE
Spreadsheet: Update statistical functions to take variadic arguments

### PR DESCRIPTION
**Pull request by water-ghosts**
This PR updates Spreadsheet's statistical functions to accept arbitrarily many arguments. These arguments can be any combination of literal values, cell references, or ranges. Previously, these functions required their arguments to be passed as a single range. 

variadic

This matches the behavior of Excel and Google Sheets.

### CODE REVIEW

File: Base/res/js/Spreadsheet/runtime.js [hunk 1]
```diff
@@ -431,38 +431,45 @@ function numericReduce(op, accumulator, cells) {
 }
 
 function numericResolve(cells) {
-    return resolve(cells).map(val => parseInt(val));
+    return resolve(cells).map(val => parseFloat(val));
```
Comment (ADKaster) [inline Base/res/js/Spreadsheet/runtime.js:434]: This seems to be a pretty significant change. Do we know why the original code was expecting all of these values to be able to be converted to an integer, and why it's ok to accept floating point values for all the callers?

Comment (water-ghosts) [reply Base/res/js/Spreadsheet/runtime.js:434]: I don't think this was a conscious decision, but probably a bug / oversight in the original code. There were no tests that checked or depended on this behavior, and even simple functions like `sum` and `min` gave the wrong results if any inputs were floats.

Comment (alimpfard) [reply Base/res/js/Spreadsheet/runtime.js:434]: (please split this patch to a separate commit)

**Review by alimpfard — COMMENTED** [review]
Changes lgtm, just split that one bugfix into its own commit.

**Pull request merged**